src/context/IntlContext.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { IntlProvider } from "react-intl";
+import App from "../App";
+import Spanish from "../lang/es-US.json";
+import English from "../lang/en-US.json";
+
+export const Context = React.createContext();
+
+// detect the language preference of the user's browser
+export const locale =
+  window.navigator.userLanguage || window.navigator.language;
+
+let messages;
+if (locale === "en-US") {
+  messages = English;
+} else {
+  messages = Spanish;
+}
```

Comment:
❓ Is it possible to extract these lines into a hook called `useLocale`, or something like that?

Reply:
Addressed:  082ccb0f91519e5b421b33e7e7453b542a2330c3